osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -46,6 +48,12 @@ protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo s
             countMiss = score.Statistics.GetValueOrDefault(HitResult.Miss);
             effectiveMissCount = calculateEffectiveMissCount(osuAttributes);
 
+            if (!usingClassicSliderAccuracy)
+            {
+                countSliderEndsDropped = osuAttributes.SliderCount - score.Statistics.GetValueOrDefault(HitResult.SliderTailHit);
+                countLargeTickMiss = score.Statistics.GetValueOrDefault(HitResult.LargeTickMiss);
+            }
+
```

Comment:
```suggestion
            if (!usingClassicSliderAccuracy)
            {
                countSliderEndsDropped = osuAttributes.SliderCount - score.Statistics.GetValueOrDefault(HitResult.SliderTailHit);
                countLargeTickMiss = score.Statistics.GetValueOrDefault(HitResult.LargeTickMiss);
            }

            effectiveMissCount = calculateEffectiveMissCount(osuAttributes);
```
Effective misscount should not be calculated before that info is available

Reply:
fixed, thanks!